### ISSUE TITLE
Is it possible to add multiple images side by side?

### ISSUE THREAD

**Issue by gerroon** [TriliumNext/trilium#977]
HI

Whenever I add an image I can only add them as a new line. However I am very much interested in being able to tile images side by side. Is this possible in text notes?

**Comment by zadam** [comment]
Yes, with the help of a table - simply create a table of one row and e.g. 3 columns and you can paste images into the columns.

**Comment by gerroon** [comment]
@zadam 
Ok that is a work around but not that awesome unfrotunately. I see two issues

- When I click on an empty space in the page first image gets white background

- The spacing between them is not adjustable. Would it be possible to resize the cells to the borders oft he images?

Please see the image below.

https://i.imgur.com/J2UvkvA.png

**Comment by zadam** [comment]
I think CKEditor will soon get some updates on the table support but I'm not sure about specifics.

I imagine one day CKEditor will support side by side images without tables but  today is not the day ...

**Comment by gerroon** [comment]
How about the white background problem? Do you know a solution?

**Comment by zadam** [comment]
Well, that's the table cell focus indicator, that's normal behavior ...

**Comment by meichthys** [comment]
[Trilium has entered maintenance mode](https://github.com/zadam/trilium/issues/4620#issuecomment-2094495698). Future enhancements will be addressed in TrilumNext: https://github.com/TriliumNext/trilium/issues/4991

**Comment by meichthys** [comment]
This is now possible using the in-line or other image formatting features in the built in editor:
Image

<img width="395" height="177" alt="Image" src="https://github.com/user-attachments/assets/bb460472-2871-4022-b0bd-20f10f25624d" />

Just select the image and choose 'in-line' - this will allow you to add any number of images on the same line.
